test/unit_tests/unittest_monitoring_frame_msg.cpp

```diff
@@ -214,7 +214,7 @@ TEST(MonitoringFrameMsgPrintTest, testPrintMessageSuccess)
   monitoring_frame::Message msg(TenthOfDegree(1234), TenthOfDegree(56), 78, { 45, 44, 43, 42 });
   EXPECT_EQ(fmt::format("{}", msg),
             "monitoring_frame::Message(fromTheta = 123.4 deg, resolution = 5.6 deg, scanCounter = 78, "
-            "measures = {45.0, 44.0, 43.0, 42.0}, intensities = {}, diagnostics = {})");
+            "measures = {45, 44, 43, 42}, intensities = {}, diagnostics = {})");
```

Comment:
This is due to different formatting of floating point numbers in libfmt version 4. At this point we probably have to distinguish between the melodic and noetic build later.